source/quickstart.txt

```diff
@@ -80,6 +80,10 @@ Setup
       The number of nodes in the destination replica set does not have
       to equal the number of nodes in the source replica set.
 
+      .. important:: 
```

Comment:
for internal reviewer: Do you think there's a better way to format/include this information in this section? Product specified that it'd be best here, but the way that the copy flows seems a bit clunky.

Reply:
I understand your concern, but I can't think of a better way to present in. The copy definitely is important and that placement makes sense to me even if adds some visual noise due to multiple admonitions in the same step. I say go with it. Thanks!

Reply:
Gotcha, thanks!